app/src/js/components/Collections/list.js

```diff
@@ -107,7 +102,7 @@ class CollectionList extends React.Component {
 
           <List
             list={list}
-            data={data}
+            data={list.data}
```

Comment:
Not a huge thing, but I believe we use list.data as the default for the data prop, so we should be able to exclude this prop